src/elfpy/pricing_models/base.py

```diff
@@ -341,22 +382,28 @@ def check_input_assertions(
     ):
         """Applies a set of assertions to the input of a trading function."""
 
-        assert quantity.amount > 0, (
-            "pricing_models.check_input_assertions: ERROR: " f"expected quantity.amount > 0, not {quantity.amount}!"
+        assert quantity.amount >= 1e-18, (
+            "pricing_models.check_input_assertions: ERROR: "
+            f"expected quantity.amount >= 1e-18, not {quantity.amount}!"
         )
-        assert market_state.share_reserves >= 0, (
+        assert market_state.share_reserves >= 1e-18, (
             "pricing_models.check_input_assertions: ERROR: "
-            f"expected share_reserves >= 0, not {market_state.share_reserves}!"
+            f"expected share_reserves >= 1e-18, not {market_state.share_reserves}!"
```

Comment:
Why can the bond_reserves == 0 but not share_reserves?

Reply:
It's an artifact of the way that we bootstrap the markets (we initialize the market with some shares and the bonds are 0 for a small period of time). To be honest, I don't really see the rationale for the current method of bootstrapping instead of just having the AMM solve for the virtual bond reserves required to hit a target APR, but I didn't want to rock the boat.

Reply:
I see, makes sense. I agree it's not ideal, but it's the [boat we're in](https://youtu.be/_uHXS36LKNE).